sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -130,9 +131,12 @@ export class Pipeline {
  * @interface StoragePipelineOptions
  */
 export interface StoragePipelineOptions {
+  /**
+   * Options to configure a proxy for outgoing requests.
+   */
```

Comment:
minor: type and memberof

Reply:
minor: For types re-exported from core, how will the docs looks like? Can user easily find doc link for these references?

---
In reply to: [339410301](https://github.com/Azure/azure-sdk-for-js/pull/5831#discussion_r339410301) [](ancestors = 339410301)

Reply:
Am not sure, I'll check with my team and get back to you